keras/src/backend/mlx/core.py

```diff
@@ -211,6 +209,10 @@ def vectorized_map(function, elements):
 def scatter(indices, values, shape):
     indices = convert_to_tensor(indices)
     values = convert_to_tensor(values)
+    if values.dtype == mx.int64:
```

Comment:
Temporary fix for until scatter bug is resolved in mlx. See [mlx_backend](https://github.com/keras-team/keras/issues/19571#issue-2254529820) .